### ISSUE TITLE
Merge Cases with options to merge tasks

### ISSUE THREAD

**Issue by david-neal-ms** [TheHive-Project/TheHive#1074]
Feature Request

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | RedHat
| OS version (client)       | Seven
| TheHive version   | 3.2.1-1
| Package Type              | From source
| Browser type & version    | Chrome

### Problem Description
Currently when merging two cases all tasks are also merged. If a case has lots of tasks, it makes the merged case quite hard to follow. Especially if the cases had duplicate tasks.

### Possible Solutions
When merging cases, prompt the user if they would like to merge tasks too. Response options could be:
                    1. Yes
                    2. No
                    3. Yes, but only ones with task log entries.

**Comment by nadouani** [comment]
Case merging has to be improved with a lots of options the user could in fact customize.

**Comment by veeral-patel** [comment]
@david-neal-ms How would you suggest merging tasks? From a UX perspective, it seems difficult to make it work well, but let me know what your thoughts are.

**Comment by nadouani** [comment]
I would show a form to customize the merge options on the merge dialog where we select the case to merge
